tests/base/base.test.js

```diff
@@ -0,0 +1,30 @@
+import { describe, it, expect } from 'vitest';
+import { ESLint } from 'eslint';
+
+describe('base config', () => {
+  it('should allow spreading of array elements', async () => {
+    // arrange
+    const linter = new ESLint();
+    const fileName = 'tests/base/array-spread.js';
+
+    // act
+    const [result] = await linter.lintFiles([fileName]);
+
+    // assert
+    expect(result.warningCount).toBe(0);
+    expect(result.messages.some((message) => message.ruleId === 'array-func/prefer-array-from')).toBe(false);
+  });
+
+  it('should allow up to 500 lines in a js-file', async () => {
+    // arrange
+    const linter = new ESLint();
+    const fileName = 'tests/base/max-lines.js';
+
+    // act
+    const [result] = await linter.lintFiles([fileName]);
+
+    // assert
+    expect(result.warningCount).toBe(0);
+    expect(result.messages.some((message) => message.ruleId === 'max-lines')).toBe(false);
```

Comment:
maybe also a file with over 500 where we expect the rule to step in?

Reply:
You could also create the file programmatically (by creating valid code and duplicating the line x times or by downloading the content of another file (maybe a GitHub gist)) and remove it after the test. But that's not necessary.